Python/cp/maxsum_subarray_python.py

```diff
@@ -0,0 +1,43 @@
+#time complexity : O(nlogn)
+
+# Returns sum of maxium sum subarray in arr[l..h]
+def maxSumSubArray(arr, low, high) :
+    # Base Case
+    if (low == high) :
+        return arr[low]
+    # Find middle element
+    mid = (low + high) // 2

```

Comment:
@poojarathore30 use `mid = low + ((high - low) / 2)` this will avoid overflow conditon.